packages/playground/website/src/lib/make-blueprint.tsx

```diff
@@ -17,11 +17,16 @@ export function makeBlueprint(options: MakeBlueprintOptions): Blueprint {
 			wp: options.wp as any,
 		},
 		steps: [
-			{
-				step: 'login',
-				username: 'admin',
-				password: 'password',
-			},
+			options.import
```

Comment:
I'm assuming this is not a problem for you, that you will address the type to add the import flag, but if not, I can try and add it in if you'd like.

Reply:
Yes, I updated the interface `MakeBlueprintOptions` to include the new `import` option.